server/graphql/v1/mutations/expenses.js

```diff
@@ -671,6 +672,20 @@ export async function payExpense(req, args) {
     );
   }
 
+  // If the amount of the expense is larger than $1000 enforce 2FA if the user has it turned on
+  const acceptedPayoutMethods =
+    payoutMethodType === PayoutMethodTypes.PAYPAL || payoutMethodType === PayoutMethodTypes.BANK_ACCOUNT;
+  const hostHasPayout2FAEnabled = get(host, 'settings.payoutsTwoFactorAuth.enabled', null);
+  const hostPayout2FAExpenseAmount = get(host, 'settings.payoutsTwoFactorAuth.expenseAmount', null);
+  if (
+    acceptedPayoutMethods &&
+    !args.forceManual &&
+    hostHasPayout2FAEnabled &&
+    expense.amount >= hostPayout2FAExpenseAmount
+  ) {
+    enforceTwoFactorAuthOnPayouts(req, args.twoFactorAuthenticatorCode);
```

Comment:
Even if it's verbose, avoiding abbreviations is generally preferable: `enforceTwoFactorAuthenticationOnPayouts`